docs/CHANGELOG.asciidoc

```diff
@@ -78,6 +78,32 @@ To review the deprecations in previous versions, refer to the following:
 
 <<deprecations-8.0.0-rc1,8.0.0-rc1>> | <<deprecations-8.0.0-alpha1,8.0.0-alpha1>>
 
+[float]
+[[known-issue-8.0.0]]
+=== Known issue
+
+[discrete]
+[[known-issue-123550]]
+.Importing and copying saved objects causes weak links to break
+[%collapsible]
+====
+*Details* +
+When you add weak links in saved objects, for example, a dashboard may include a Markdown panel that contains a relative URL to
```

Comment:
I'm not a native speaker, but this first sentence sounds a bit weird. Does it mean something like `It's possible to add "weak links" in saved objects. For example a dashboard may include a Markdown panel that contains a relative URL to another dashboard` ?

Reply:
Good catch, this got changed when it was moved and I agree it doesn't make sense.

I'll change it to this:

"Kibana supports weak links in some saved objects. For example, a dashboard may include a Markdown panel that contains a relative URL to
another dashboard. Weak links are defined by free text, _not_ the saved object's relationships, and can break if **both** of the following
conditions are true:"